xml/System.Data.Common/DbException.xml

```diff
@@ -345,8 +354,13 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>
+          <para> For database providers which support it, contains a standard SQL 5-character return code indicating the success or failure of the database operation. The first 2 characters represent the <strong>class</strong> of the return code (e.g. error, success), while the last 3 characters represent the <strong>subclass</strong>, allowing detection of error scenarios in a database-portable way.
```

Comment:
do tags like <strong> really work here? @carlossanlop

Reply:
They might. Let's see in the preview.

Reply:
Yes, visible and looking good: https://review.docs.microsoft.com/en-us/dotnet/api/system.data.common.dbexception.sqlstate?view=net-5.0&branch=pr-en-us-4792#System_Data_Common_DbException_SqlState